### ISSUE TITLE
Delete archive version in the UI

### ISSUE THREAD

**Issue by sylmorin-gbif** [gbif/ipt#1508]
We would like to delete a specific version of a resource, directly in the UI.

Ex: when we generated a corrupted version of resource

It could be available in this screen:
![image](https://user-images.githubusercontent.com/56630013/96749643-5cbb4e00-13cb-11eb-9971-dc09d8be702e.png)

Only the admin could perform this action.
It won't be possible to delete the latest version - only the archived versions.

Question: should we delete only the DwC archive file (like when Archival Mode is not activated) or the whole archive version (DwC archive, EML, RTF, and version in the history)?

**Comment by timrobertson100** [comment]
This seems sensible to add to the UI.

I expect a manual procedure can be documented in the meantime. I'll explore that now

**Comment by timrobertson100** [comment]
For a workaround, you can do the following.

- Imagine you wish to remove version 1.2 of resouce `aaa`.
- find the resource `aaa` directory in the ipt data directory
- open the `resource.xml`
- towards the bottom locate the `versionhistory` element containing the `<version>1.2</version>`
- comment out the `versionhistory` element 
- restart the IPT

Here is a snippet of the XML as an example:

```
    <versionhistory>
      <version>1.3</version>
      <released>2020-11-11 15:43:14.207 UTC</released>
      <status>UNRESERVED</status>
      <publicationStatus>PRIVATE</publicationStatus>
      <modifiedBy reference="../../../creator"/>
      <changeSummary>d</changeSummary>
      <recordsPublished>1</recordsPublished>
      <recordsByExtension>
        <entry>
          <string>http://rs.tdwg.org/dwc/terms/Occurrence</string>
          <int>1</int>
        </entry>
      </recordsByExtension>
    </versionhistory>
    <!--
     REMOVING VERSION 1.2 AS IT WAS CORRUPT
      versionhistory>
      <version>1.2</version>
      <released>2020-11-11 15:43:05.772 UTC</released>
      <status>UNRESERVED</status>
      <publicationStatus>PRIVATE</publicationStatus>
      <modifiedBy reference="../../../creator"/>
      <changeSummary>c</changeSummary>
      <recordsPublished>1</recordsPublished>
      <recordsByExtension>
        <entry>
          <string>http://rs.tdwg.org/dwc/terms/Occurrence</string>
          <int>1</int>
        </entry>
      </recordsByExtension>
    </versionhistory-->
    <versionhistory>
      <version>1.1</version>
      <released>2020-11-11 15:42:53.469 UTC</released>
      <status>UNRESERVED</status>
      <publicationStatus>PRIVATE</publicationStatus>
      <modifiedBy reference="../../../creator"/>
      <changeSummary>b</changeSummary>
      <recordsPublished>1</recordsPublished>
      <recordsByExtension>
        <entry>
          <string>http://rs.tdwg.org/dwc/terms/Occurrence</string>
          <int>1</int>
        </entry>
      </recordsByExtension>
    </versionhistory>
```

**Comment by sylmorin-gbif** [comment]
Totally agree with the sensible part of this feature.
That's why, I restricted it to admin users (who can also do much more damages by modifying manually the resource.xml file...)

My concern was rather about the citations.
We could delete only the DwC archive file (to save disk space, or remove bad data) instead of the whole version.
All the code is the same, we just have to switch backend java method called (the "delete only DwC archive" method already exist)
 
Here is the PR, with the code I did some days ago: https://github.com/gbif/ipt/pull/1513
